### PR TITLE
clusterlogging pkg: lokistack handler added

### PR DESCRIPTION
**Pull request by elenagerman**

### CODE REVIEW

**Comment by elenagerman** [comment]
@trewest @kononovn @achuzhoy @cdvultur PTAL

File: pkg/clusterlogging/lokistack.go [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+
+// WithStorageClassName sets the lokiStack operator's storage class name configuration.
+func (builder *LokiStackBuilder) WithStorageClassName(
+	storageClassName string) *LokiStackBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof(
+		"Setting lokiStack %s in namespace %s with the the storage class name config: %v",
```
Comment (trewest) [inline pkg/clusterlogging/lokistack.go:268]:
```suggestion
		"Setting lokiStack %s in namespace %s with the storage class name: %s",
```

File: pkg/clusterlogging/lokistack.go [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+// WithTenants sets the lokiStack operator's tenants configuration.
+func (builder *LokiStackBuilder) WithTenants(
+	tenants lokiv1.TenantsSpec) *LokiStackBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof(
+		"Setting lokiStack %s in namespace %s with the the tenants config: %v",
```
Comment (trewest) [inline pkg/clusterlogging/lokistack.go:292]:
```suggestion
		"Setting lokiStack %s in namespace %s with the tenants config: %v",
```

File: pkg/clusterlogging/lokistack.go [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+
+// WithRules sets the lokiStack operator's rules configuration.
+func (builder *LokiStackBuilder) WithRules(
+	rules lokiv1.RulesSpec) *LokiStackBuilder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof(
+		"Setting lokiStack %s in namespace %s with the the rules config: %v",
```
Comment (trewest) [inline pkg/clusterlogging/lokistack.go:308]:
```suggestion
		"Setting lokiStack %s in namespace %s with the rules config: %v",
```

**Review by trewest — CHANGES_REQUESTED** [review]
Looks good just a few cosmetic changes

**Comment by elenagerman** [comment]
> Looks good just a few cosmetic changes

Thanks!

**Comment by elenagerman** [comment]
@trewest @kononovn @achuzhoy @cdvultur PTAL

File: pkg/clusterlogging/lokistack.go [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+	}
+
+	if name == "" {
+		glog.V(100).Infof("The name of the lokiStack is empty")
+
+		builder.errorMsg = "lokiStack 'name' cannot be empty"
+	}
+
+	if nsname == "" {
+		glog.V(100).Infof("The nsname of the elasticsearch is empty")
```
Comment (trewest) [inline pkg/clusterlogging/lokistack.go:58]:
```suggestion
		glog.V(100).Infof("The nsname of the lokiStack is empty")
```

**Review by trewest — APPROVED** [review]
/lgtm

File: pkg/clusterlogging/lokistack.go [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+		builder.errorMsg = "lokiStack 'nsname' cannot be empty"
+	}
+
+	return builder
+}
+
+// PullLokiStack retrieves an existing lokiStack object from the cluster.
+func PullLokiStack(apiClient *clients.Settings, name, nsname string) (*LokiStackBuilder, error) {
+	glog.V(100).Infof(
+		"Pulling lokiStack object name:%s in namespace: %s", name, nsname)
```
Comment (achuzhoy) [inline pkg/clusterlogging/lokistack.go:69]: nit: 
```suggestion
		"Pulling lokiStack object name: %s in namespace: %s", name, nsname)
```

File: pkg/clusterlogging/lokistack.go [hunk 6]
```diff
@@ -0,0 +1,362 @@
...
+	Definition *lokiv1.LokiStack
+	// Created lokiStack object on the cluster.
+	Object *lokiv1.LokiStack
+	// api client to interact with the cluster.
+	apiClient goclient.Client
+	// errorMsg is processed before lokiStack object is created.
+	errorMsg string
+}
+
+// NewLokiStackBuilder method creates new instance of builder.
```
Comment (achuzhoy) [inline pkg/clusterlogging/lokistack.go:29]:
```suggestion
// NewLokiStackBuilder creates new instance of builder.
```

**Review by achuzhoy — APPROVED** [review]

**Pull request merged**